### PR TITLE
Msi

### PR DESCRIPTION
**Pull request by timosong**
Added msisensor to the variant caller module and made appropriate changes. 
This should also generate a new directory called "analysis" for future output. 

i hope this works

### CODE REVIEW

**Comment by allanbolipata** [comment]
add to whitelist

**Comment by timosong** [comment]
test this, please

File: setup/bin/roslin_portal_helper.py [hunk 1]
```diff
@@ -77,17 +77,21 @@ def generate_clinical_data(clinical_data_path,clinical_output_path,coverage_valu
...
 	with open(clinical_data_path) as input_file, open(clinical_output_path,'w') as output_file:
 		writer = csv.writer(output_file, lineterminator='\n',dialect='excel-tab')
 		reader = csv.reader(input_file,dialect='excel-tab')
 		clinical_data = []
 		row = reader.next()
 		row.append('SAMPLE_COVERAGE')
+		row.append('MSI_SCORE')
 		clinical_data.append(row)
 		for row in reader:
+			print row
```
Comment (nikhil) [inline setup/bin/roslin_portal_helper.py]: Seems like an artifact, please remove the print statement.

File: setup/cwl/project-workflow-sv.cwl [hunk 2]
```diff
@@ -6,9 +6,9 @@ $namespaces:
   doap: http://usefulinc.com/ns/doap#
 
 $schemas:
-- http://dublincore.org/2012/06/14/dcterms.rdf
-- http://xmlns.com/foaf/spec/20140114.rdf
-- http://usefulinc.com/ns/doap#
+- file:///ifs/work/pi/roslin-test/targeted-variants/200/roslin-core/2.0.0/schemas/dcterms.rdf
```
Comment (nikhil) [inline setup/cwl/project-workflow-sv.cwl]: Just to be consistent with the rest of our cwls can you revert the schemas to:

- http://dublincore.org/2012/06/14/dcterms.rdf
- http://xmlns.com/foaf/spec/20140114.rdf
- http://usefulinc.com/ns/doap#

File: setup/cwl/project-workflow-sv.cwl [hunk 3]
```diff
@@ -456,6 +466,22 @@ steps:
 
     out: [ as_metrics, hs_metrics, insert_metrics, insert_pdf, per_target_coverage, qual_metrics, qual_pdf, doc_basecounts, gcbias_pdf, gcbias_metrics, gcbias_summary, qc_files]
 
+<<<<<<< HEAD
```
Comment (nikhil) [inline setup/cwl/project-workflow-sv.cwl]: This segment should be removed

File: setup/cwl/project-workflow.cwl [hunk 4]
```diff
@@ -6,9 +6,9 @@ $namespaces:
   doap: http://usefulinc.com/ns/doap#
 
 $schemas:
-- http://dublincore.org/2012/06/14/dcterms.rdf
-- http://xmlns.com/foaf/spec/20140114.rdf
-- http://usefulinc.com/ns/doap#
+- file:///ifs/work/pi/roslin-test/targeted-variants/200/roslin-core/2.0.0/schemas/dcterms.rdf
```
Comment (nikhil) [inline setup/cwl/project-workflow.cwl]: Same as workflow-sv
Just to be consistent with the rest of our cwls can you revert the schemas to:

- http://dublincore.org/2012/06/14/dcterms.rdf
- http://xmlns.com/foaf/spec/20140114.rdf
- http://usefulinc.com/ns/doap#

**Review by nikhil — CHANGES_REQUESTED** [review]
Left a few comments.

File: setup/bin/roslin_portal_helper.py [hunk 5]
```diff
@@ -312,11 +316,12 @@ def wait_for_job_to_finish(bjob_id,name):
 	job_string = name+' [' + bjob_id+']'
 	logger.info("Monitoring "+job_string)
 	bjob_command = "bjobs " + bjob_id + " | awk '{print $3}' | tail -1"
-	job_done = False
+	# job_done = False  #ts
+	job_done = True
```
[the remark this answers is not in the document]
Comment (timosong) [reply setup/bin/roslin_portal_helper.py]: I don't remember making this change.. Changing it back to `job_done = False`

**Comment by timosong** [comment]
test this, please

**Comment by timosong** [comment]
test this, please

**Comment by timosong** [comment]
test this, please

**Comment by timosong** [comment]
test this, please

**Review by nikhil — APPROVED** [review]

**Comment by ckandoth** [comment]
With all the CWL changes in `2.5.0`, it will be easier to create a new branch, rather than try to merge this one. I'll close this PR, but the code will remain in the `msi` branch, till we can work it into a `2.5.*` release

**Pull request closed without merging**